flight/odlc_boundaries/search_path.py

```diff
@@ -0,0 +1,204 @@
+"""
+Functions for generating search paths to cover an area for finding the standard odlc objects
+"""
+
+import logging
+import asyncio
+import utm
+from shapely.geometry import Polygon
+from execute import move_to
+from mavsdk import System
+
+
+def latlon_to_utm(coords: dict[str, float]) -> dict[str, float]:
+    """
+    Converts latlon coordinates to utm coordinates and adds the data to the dictionary
+
+    Parameters
+    ----------
+    coords : Dict[str, float]
+        A dictionary containing lat long coordinates
+
+    Returns
+    -------
+    coords : Dict[str, float]
+        An updated dictionary with additional keys and values with utm data
+    """
+
+    utm_coords: tuple[float, float, int, str] = utm.from_latlon(
+        coords["latitude"], coords["longitude"]
+    )
+    coords["utm_x"] = utm_coords[0]
+    coords["utm_y"] = utm_coords[1]
+    return coords
+
+
+def all_latlon_to_utm(list_of_coords: list[dict[str, float]]) -> list[dict[str, float]]:
+    """
+    Converts a list of dictionaries with latlon data to add utm data
+
+    Parameters
+    ----------
+    list_of_coords : list[dict[str, union[float, int, str]]]
+        A list of dictionaries that contain lat long data
+
+    Returns
+    -------
+    list_of_coords : [dict[str, union[float, int, str]]]
+        An updated list of dictionaries with added utm data
+    """
+    coord: int
+    for coord, _ in enumerate(list_of_coords):
+        list_of_coords[coord] = latlon_to_utm(list_of_coords[coord])
```

Comment:
You don't really need to use `enumerate` here since you aren't using the item attribute, only the index.
```suggestion
    for coord in range(len(list_of_coords)):
        list_of_coords[coord] = latlon_to_utm(list_of_coords[coord])
```